content/_data/changelogs/weekly.yml

```diff
@@ -1038,6 +1038,34 @@
       message: >
         Internal: Updated parent POM; Jenkins core now requires Maven 3.3.9 or newer to build.
       pull: 2985
+- version: "2.77"
+  date: 2017-09-03
+  changes:
+    - type: rfe
+      message: >
+        Default the built-in Jenkins Update Center URL to <code>https://updates.jenkins.io</code> instead of obsolete HTTP endpoint.
+      pull: 2996
+    - type: bug
+      message: >
+        Fix problem with auto upgrade when using custom <code>JENKINS_HOME</code> on Windows.
+      issue: 13153
+      pull: 2992
+    - type: bug
+      message: >
+        Administrative monitor did not detect when Tomcat's URL escaping does not permit forward slashes.
+      issue: 31068
+      pull: 2977
+    - type: bug
+      message: >
+        Fix broken UI for users with Discover permission when renaming a job.
+      issue: 41637
+      pull: 2910
+    - type: rfe
+      message: >
+        Internal: Avoid code duplication using default methods.
```

Comment:
IMHO not required. It does not change the API or whatever, so it's interesting only to core contributors as an encourangement to do the same

Reply:
Personally, I have a lower threshold for 'internal' messages. Who knows when they become relevant to API users. My hope is the prefix just makes users gloss over them.